### PR TITLE
Add missing extract_ps and subs_epi intrinsics

### PR DESCRIPTION
**Pull request by danlark1**
These intrinsics likely were missed at the time -- they are an extension of the similar ones

### CODE REVIEW

File: sse2neon.h [hunk 1]
```diff
@@ -3568,6 +3581,16 @@ FORCE_INLINE __m128i _mm_minpos_epu16(__m128i a)
...
+// Inserts the least significant 8 bits of b into the selected 8-bit integer
+// of a.
+// FORCE_INLINE __m128 _mm_insert_ps(__m128 a, int b,
+//                                   __constrange(0,16) int imm)
+#define _mm_insert_ps(a, b, imm)                                  \
+    __extension__({                                               \
+        vreinterpretq_m128_s8(                                    \
+            vsetq_lane_s8((b), vreinterpretq_s8_m128(a), (imm))); \
+    })
+
```
Comment (marktwtn) [inline sse2neon.h:3593]: The behaviour of `_mm_insert_ps` is more complicated than `vsetq_lane_s8`.
Please check: [_mm_insert_ps](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_insert_ps&expand=5265,5477,2435,5778,3148)

The implementation should be rewritten or left to the future pull request.

File: sse2neon.h [hunk 2]
```diff
@@ -1885,6 +1885,19 @@ FORCE_INLINE __m128i _mm_subs_epu8(__m128i a, __m128i b)
         vqsubq_u8(vreinterpretq_u8_m128i(a), vreinterpretq_u8_m128i(b)));
 }
 
+// Subtracts the 16 signed 8-bit integers of b from the 16 signed 8-bit integers
+// of a and saturates.
+//
+//   r0 := SignedSaturate(a0 - b0)
+//   r1 := SignedSaturate(a1 - b1)
+//   ...
+//   r15 := SignedSaturate(a15 - b15)
```
[the remark this answers is not in the document]
Comment (danlark1) [reply sse2neon.h:1894]: Done

File: sse2neon.h (hunk 1 above)
Comment (danlark1) [reply sse2neon.h]: Agree, done

File: sse2neon.h [hunk 3]
```diff
@@ -3592,6 +3615,11 @@ FORCE_INLINE __m128i _mm_minpos_epu16(__m128i a)
 #define _mm_extract_epi32(a, imm) \
     vgetq_lane_s32(vreinterpretq_s32_m128i(a), (imm))
 
+// Extracts the selected signed or unsigned 32-bit integer from a and zero
+// extends.
```
[the remark this answers is not in the document]
Comment (danlark1) [reply sse2neon.h]: Done

File: sse2neon.h [hunk 4]
```diff
@@ -1885,13 +1885,28 @@ FORCE_INLINE __m128i _mm_subs_epu8(__m128i a, __m128i b)
...
 }
 
+// Subtracts the 16 signed 8-bit integers of b from the 16 signed 8-bit integers
+// of a and saturates.
+//
+//   r0 := SignedSaturate(a0 - b0)
+//   r1 := SignedSaturate(a1 - b1)
+//   ...
+//   r15 := SignedSaturate(a15 - b15)
+// https://technet.microsoft.com/en-us/subscriptions/by7kzks1(v=vs.90)
```
Comment (marktwtn) [inline sse2neon.h:1895]: Please add a blank line before the link, thanks~

File: sse2neon.h [hunk 5]
```diff
@@ -1885,13 +1885,28 @@ FORCE_INLINE __m128i _mm_subs_epu8(__m128i a, __m128i b)
...
+}
+
 // Subtracts the 8 signed 16-bit integers of b from the 8 signed 16-bit integers
 // of a and saturates.
 //
 //   r0 := SignedSaturate(a0 - b0)
 //   r1 := SignedSaturate(a1 - b1)
 //   ...
 //   r7 := SignedSaturate(a7 - b7)
+// https://technet.microsoft.com/en-us/subscriptions/3247z5b8(v=vs.90)
```
Comment (marktwtn) [inline sse2neon.h:1909]: Ditto.

**Comment by danlark1** [comment]
> Please rebase and squash the commits.

Done

**Pull request merged**